src/hooks/useMediaQuery.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+
+type MediaQueryHook = (query: string) => boolean;
+
+const useMediaQuery: MediaQueryHook = (query) => {
+  const [matches, setMatches] = useState(false);
```

Comment:
Really nice way to check the device good job

Reply:
code details resolved in pr year and volume

Please check my pr ing